### ISSUE TITLE
Support for Kaios 3

### ISSUE THREAD

**Issue by AnisTaluqdar** [farooqkz/chooj#52]

**Comment by farooqkz** [comment]
Unfortunately I haven't got a KaiOS 3+ phone

**Comment by minhduc-bui1** [comment]
Should be pretty easy to just change the name of manifest.webapp to manifest.webmanifest; create a file called manifest.en-US.webmanifest; and making other API changes if needed. Unfortunately none of KaiOS 3.x devices currently available can be debugged and side-loaded with third-party apps.

**Comment by farooqkz** [comment]
Yup it's easy. I said I don't have a device which is debug enabled so that I can test.

**Comment by farooqkz** [comment]
Some notes:

 - [2780](https://wiki.bananahackers.net/en/devices/nokia-2780-flip) is now available in Amazon but it seems debug locked
 - At the time of writing this note, All KaiOS 3.x phones are for US.
 - We'll need separate build config for KaiOS 3.x because the current configuration is optimized and made to work on 2.x not 3.x. Not a lot of device APIs have been used but there is no reason to transpile JS if the code is supposed to run on new Gecko on an ancient one.

**Comment by JakoDel** [comment]
> Some notes:
> 
>     * [2780](https://wiki.bananahackers.net/en/devices/nokia-2780-flip) is now available in Amazon but it seems debug locked
> 
>     * At the time of writing this note, All KaiOS 3.x phones are for US.
> 
>     * We'll need separate build config for KaiOS 3.x because the current configuration is optimized and made to work on 2.x not 3.x. Not a lot of device APIs have been used but there is no reason to transpile JS if the code is supposed to run on new Gecko rather than an ancient one.

Would this somewhat work on a kaiOS 3.0 device if I were to make just the necessary changes to the manifest? having a matrix client working would straight up replace 2.5 for me, since I'd have whatsapp and basically all message services working.
Noticed that you're away, but hopefully not completely off the grid :)
thanks for your work!
